mcp-server/carabassa_mcp/server.py

```
@@ -0,0 +1,426 @@
+def get_item(dataset_id: int, item_id: int) -> Any:
+    """Get full details of a single item, including all its tags.
+
+    Args:
+        dataset_id: Numeric dataset identifier.
+        item_id: Numeric item identifier.
+    """
+    return _get(f"/dataset/{dataset_id}/item/{item_id}")
+
+
+@mcp.tool()
+def item_exists_by_hash(dataset_id: int, hash: str) -> str:
```

Ruff (0.15.13)

[error] Function argument `hash` is shadowing a Python builtin

(A002)